### PR TITLE
Update networks supported by core services

### PR DESCRIPTION
**Pull request by KevinLu**
Rinkeby is no longe supported by Safe.

### CODE REVIEW

**Review by ademidun — APPROVED** [review]
Thanks for starting this PR @KevinLu .

I also used this opportunity to piggyback on some other changes as well.

File: learn/infrastructure/service-architecture.md [hunk 1]
```diff
@@ -4,20 +4,28 @@ description: Up-to-date overview document on how our services interact with each
...
+
+The transaction services use event indexing to keep track of transactions related to our safe contracts. There is one instance of the core services per supported network (mainnet, goerli, gnosis chain, polygon, among others).
+
+## Safe Config Service
 
 The config services keeps track of all the networks we support and therefore of all the instances of the core services that are available. In addition to this, the config service provides information regarding chain dependant variables such as RPC endpoints, gas price oracles, in which URL a given core service instance for a chain can be reached, among other things.
 
+## Safe Client Gateway
+
 The client gateway leverages the config services to determine how to reach the core service instance required for a given request. If the network is supported (which is determined by the config service, as aforementioned) the client gateway forwards the request to the given core service instance. The client gateway transforms (and sometimes aggregates) data sources from the core services, catering specifically to the UI needs of our frontend clients. This process is illustrated in Fig. 2
```
Comment (Uxio0) [inline learn/infrastructure/service-architecture.md:29]: transforms (and sometimes aggregates) -> `transforms, aggregates and caches information from Config Service and Transaction Service, optimizing data for our web and mobile clients`

**Review by Uxio0 — CHANGES_REQUESTED** [review]

File: learn/infrastructure/service-architecture.md [hunk 2]
```diff
@@ -4,20 +4,28 @@ description: Up-to-date overview document on how our services interact with each
 
 # Service Architecture
 
-Currently our whole infrastracture is constituted by mainly 3 services. Here is a list with the links to their respective repositories:
+The Safe infrastracture consists of 3 services:
```
[the remark this answers is not in the document]
Comment (KevinLu) [reply learn/infrastructure/service-architecture.md:7]: Did you want to fix this typo?

File: learn/infrastructure/service-architecture.md [hunk 3]
```diff
@@ -4,20 +4,28 @@ description: Up-to-date overview document on how our services interact with each
...
 ![Fig 1. Broad view of the backend services and their components.](<../../.gitbook/assets/Slide 16\_9 - 2.png>)
 
-The core services use event indexing to keep track of transactions related to our safe contracts. There is one instance of the core services per supported network (mainnet, rinkeby, xdai, polygon, among others).
+## Safe Transaction Service
+
+The transaction services use event indexing to keep track of transactions related to our safe contracts. There is one instance of the core services per supported network (mainnet, goerli, gnosis chain, polygon, among others).
+
+## Safe Config Service
 
 The config services keeps track of all the networks we support and therefore of all the instances of the core services that are available. In addition to this, the config service provides information regarding chain dependant variables such as RPC endpoints, gas price oracles, in which URL a given core service instance for a chain can be reached, among other things.
```
[the remark this answers is not in the document]
Comment (KevinLu) [reply learn/infrastructure/service-architecture.md:25]: Agreed.

**Review by Uxio0 — APPROVED** [review]

File: learn/infrastructure/service-architecture.md [hunk 4]
```diff
@@ -4,20 +4,28 @@ description: Up-to-date overview document on how our services interact with each
...
 
 As you can see from Fig. 1 , our full production setup is a mixture of several instances of the Safe Transaction Service (from now on referred to as "core service") orchestrated by the Safe Config Service (henceforth referred to as "config service") which are later consumed by the Safe Client Gateway (here on after referred to as "client gateway").
 
 ![Fig 1. Broad view of the backend services and their components.](<../../.gitbook/assets/Slide 16\_9 - 2.png>)
 
-The core services use event indexing to keep track of transactions related to our safe contracts. There is one instance of the core services per supported network (mainnet, rinkeby, xdai, polygon, among others).
+## Safe Transaction Service
+
+The transaction services use event indexing to keep track of transactions related to our safe contracts. There is one instance of the core services per supported network (mainnet, goerli, gnosis chain, polygon, among others).
```
[the remark this answers is not in the document]
Comment (ademidun) [reply learn/infrastructure/service-architecture.md]: I thought Goerli uses event indexing?  cc @mmv08 

[This Safe](https://app.safe.global/home?safe=gor:0xb67bF8d736b7024eaE034Fc0fBe80625823a6b76) was created on Goerli and uses [`GnosisSafeL2`](https://github.com/safe-global/safe-contracts/blob/main/contracts/SafeL2.sol#L7) as it's [contract implementation address](https://goerli.etherscan.io/address/0x3E5c63644E683549055b9Be8653de26E0B4CD36E).

See [contract creation hash](https://goerli.etherscan.io/tx/0x1bbb14cf468099c16796cebb07cf2dbb0df438f0e9a06e4fe8e56688c4e3a8d8).

[GnosisSafeL2 emits a `SafeMultiSigTransaction` Event](https://github.com/safe-global/safe-contracts/blob/186a21a74b327f17fc41217a927dea7064f74604/contracts/GnosisSafeL2.sol#L56) unlike [GnosisSafe.sol](https://github.com/safe-global/safe-contracts/blob/186a21a74b327f17fc41217a927dea7064f74604/contracts/GnosisSafe.sol).

[the remark this answers is not in the document]
Comment (mmv08) [reply learn/infrastructure/service-architecture.md]: I believe it supports both technically, but the interface uses the tracing implementation by default.

File: learn/safe-core-api/available-services.md [hunk 5]
```diff
@@ -15,7 +15,6 @@ description: This pages provides an overview of the available services
 | Avalanche                    | [https://safe-transaction-avalanche.safe.global/](https://safe-transaction-avalanche.safe.global/)       |
 | Aurora                       | [https://safe-transaction-aurora.safe.global/](https://safe-transaction-aurora.safe.global/)             |
 | Binance Smart Chain          | [https://safe-transaction-bsc.safe.global/](https://safe-transaction-bsc.safe.global/)                   |
-| Energy Web Chain             | [https://safe-transaction-ewc.safe.global/](https://safe-transaction-ewc.safe.global/)                   |
```
[the remark this answers is not in the document]
Comment (mmv08) [reply learn/safe-core-api/available-services.md:18]: We will be discontinuing our UI and API support for Energy Web Chain and Volta on Wednesday, March 22nd, 2023. If you have any questions or concerns, please reach out to the EWC team who will provide alternative support.

**Comment by KevinLu** [comment]
@ademidun I've resolved the conflict

File: learn/infrastructure/service-architecture.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Uxio0) [reply learn/infrastructure/service-architecture.md]: For Goerli we use tracing, to have a testing environment similar to mainnet. Services using trace indexing are able to see both L2 and regular Safes

**Pull request merged**